src/libs/Navigation/AppNavigator/createResponsiveStackNavigator/ThreePaneView.js

```diff
@@ -71,6 +71,7 @@ function ThreePaneView(props) {
                                     onPress={() => props.navigation.goBack()}
                                     accessibilityLabel={translate('common.close')}
                                     accessibilityRole={CONST.ACCESSIBILITY_ROLE.BUTTON}
+                                    noDragArea
```

Comment:
This change causing this [issue](https://github.com/Expensify/App/pull/24098#issuecomment-1670450138)